crates/router/src/utils/user/sample_data.rs

```diff
@@ -20,6 +20,7 @@ pub async fn generate_sample_data(
     state: &SessionState,
     req: SampleDataRequest,
     merchant_id: &id_type::MerchantId,
+    org_id: &id_type::OrganizationId,
```

Comment:
NIT: We can take the user_from_token struct instead. Not a required change though.